packages/atlas/src/providers/assets/assets.provider.tsx

```diff
@@ -171,6 +171,13 @@ export const OperatorsContextProvider: FC<PropsWithChildren> = ({ children }) =>
         const msDuration = performance.now() - startTime
         const previousTimeout = userBenchmarkTime ?? atlasConfig.storage.assetResponseTimeout
         if (msDuration > previousTimeout || msDuration < previousTimeout / 2) {
+          const newTime = (msDuration + previousTimeout) * 0.75
+          if (newTime > 2_000) {
+            SentryLogger.message('User benchmark time is too high', 'OperatorsContextProvider', 'warning', {
```

Comment:
I'm a bit afraid this might be pretty frequent. The problem is not that so many users have slow connections, but I think performance logic is flawed, basically query to get a random video returns the same video and its assets tend to fail. Am I missing smth?